docs/PowerAuth-Push-Server-1.3.0.md

```diff
@@ -83,4 +83,19 @@ Also, in several API calls, the `id` variable was renamed to `appId`, to maintai
   - `PUT /admin/app/android/update`
 - Remove Android configuration for FCM
   - `POST /admin/app/android/remove`
-  - `DELETE /admin/app/android/remove`
\ No newline at end of file
+  - `DELETE /admin/app/android/remove`
+
+## Database Dialect Configuration
+
+The latest release of PowerAuth requires configuration of database dialect.
+
+The dialect is specified using following configuration property:
+```properties
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQLDialect
```

Comment:
We are using a deprecated value in the example. Could we change it?

Reply:
Sure, good idea.